### DISCUSSION TITLE
About pre-compiled binaries

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#3002] [Q&A]
![image](https://user-images.githubusercontent.com/42310255/198889039-4ca796af-1c6a-42b2-8190-1d9d99625f68.png)

我用的vs2022，理论上应该下vc143的包才对，不过xmake还是下的vc142，这是为什么

**Comment by waruqi** [comment]
这边测试 本地 vs2022 的 toolset 14.32.31326    而 build_artifacts 上是 14.33.31629，子版本更新，所以无法匹配，你可以升级下本地 v2022 试试。。
